src/main/daml/Daml/Finance/Settlement/Instruction.daml

```diff
@@ -65,7 +65,13 @@ template Instruction
         assertMsg (show others <> " must be a subset of actors") $ others `S.isSubsetOf` actors
       assertIsCustodian : Party -> Update ()
       assertIsCustodian other =
-        assertMsg ("custodian must be " <> show other) $ routedStep.custodian == other
+        assertMsg ("custodian must match " <> show other) $ routedStep.custodian == other
+      assertIsReceiver : Party -> Update ()
+      assertIsReceiver other =
+        assertMsg ("receiver must match " <> show other) $ routedStep.receiver == other
```

Comment:
Perhaps `party` is cleaner than `other` ? (can also be applied to the relating functions)

```suggestion
      assertIsReceiver party =
        assertMsg ("receiver must match " <> show party) $ routedStep.receiver == party
```

Reply:
I renamed `other` to `party` (for `assertIsReceiver`, `assertIsSender`, and `assertIsCustodian`).